src/ui/components/ExperimentBrowserPage.tsx

```diff
@@ -24,13 +24,146 @@ enum Environment {
   STAGE = "stage",
 }
 
+type ToastVariant = "success" | "danger";
+
+interface AddToastParams {
+  message: string;
+  variant: ToastVariant;
+  autohide?: boolean;
+}
```

Comment:
We have these [here](https://github.com/mozilla-extensions/nimbus-devtools/blob/main/src/ui/hooks/useToasts.ts). We can mark them for export and import them here.